packages/website/components/Playground/SandpackRenderer.tsx

```diff
@@ -100,6 +100,8 @@ export function SandpackRenderer({
           '@contentful/f36-navlist': '^4.1.0-alpha.0', // Remove when added to f36-components
           '@contentful/f36-tokens': '^4.0.0',
           '@contentful/f36-icons': '^4.0.0',
+          '@contentful/f36-icons-alpha':
+            'npm:@contentful/f36-icons@^5.0.0-alpha.26',
```

Comment:
The version isn't a biggie but having the range can be an issue because it can point to the first version which wasn't so mature.

```suggestion
            'npm:@contentful/f36-icons@5.0.0-alpha.30',
```